# Created on Mon. 23 Sep. 2019

string id     # string labeled id

geometry_msgs/Point position     # point labeled position

# contains position of a point in free space
  # float64 x
  # float64 y
  # float64 z

